test/avocado/machineslib.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
could you please move this library to ``testlib_avocado`` module

Reply:
Sure.